src/helpers/formatAmount.ts

```diff
@@ -125,9 +125,21 @@ const convertToBigNumber = (
  * @returns {string} Formatted token amount string with optional token code
  *
  * @example
- * formatTokenAmount("1234.56"); // Returns "1,234.56" (based on device settings)
+ * // US format (dot decimal, comma thousands)
+ * formatTokenAmount("1234.56"); // Returns "1,234.56"
+ * formatTokenAmount("1234567.89"); // Returns "1,234,567.89"
+ * formatTokenAmount("0.0000012345"); // Returns "0.0000012345"
+ * formatTokenAmount("9007199254740992.123456789012345"); // Returns "9,007,199,254,740,992.123456789012345"
+ *
+ * // European format (comma decimal, dot thousands) - when device locale is set to European
+ * formatTokenAmount("1234,56"); // Returns "1.234,56" (based on device settings)
+ * formatTokenAmount("1234567,89"); // Returns "1.234.567,89" (with thousands separators)
+ * formatTokenAmount("0,0000012345"); // Returns "0,0000012345" (preserves precision)
```

Comment:
shouldn't this function always take amount with dot notation? meaning that it takes a numeric value and returns a displayable string amount